app/models/concerns/normalize_column.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module NormalizeColumn
+  extend ActiveSupport::Concern
+
+  included do
+    def self.normalize_column(column_name)
+      before_validation -> do # rubocop:disable Style/Lambda
```

Comment:
I've opted for this syntax as opposed to:

```ruby
before_validation lambda {
  # ...
}
```

or

```ruby
before_validation(lambda do
  # ....
end)
```

This syntax doesn't work:
```ruby
before_validation lambda do
  # ...
end
```